src/main/java/com/synopsys/integration/alert/channel/jira/descriptor/JiraDescriptor.java

```diff
@@ -30,9 +30,13 @@
 
 @Component
 public class JiraDescriptor extends ChannelDescriptor {
-    public static final String KEY_JIRA_URL = "jira.url";
-    public static final String KEY_JIRA_USERNAME = "jira.user.name";
-    public static final String KEY_JIRA_ACCESS_TOKEN = "jira.access.token";
+    public static final String KEY_JIRA_URL = "jira.cloud.url";
+    public static final String KEY_JIRA_USERNAME = "jira.cloud.user.name";
+    public static final String KEY_JIRA_ACCESS_TOKEN = "jira.cloud.access.token";
+
+    public static final String KEY_ADD_COMMENTS = "channel.cloud.jira.add.comments";
+    public static final String KEY_ISSUE_CREATOR = "channel.cloud.jira.issue.creator";
+    public static final String KEY_JIRA_PROJECT_NAME = "channel.cloud.jira.project.name";
```

Comment:
Why are these field cloud.jira when the other fields have jira.cloud?

Reply:
It will be fixed